settings.gradle

```diff
@@ -75,6 +75,7 @@ List projects = [
   'qa:smoke-test-ingest-with-all-dependencies',
   'qa:smoke-test-ingest-disabled',
   'qa:smoke-test-multinode',
+  'qa:smoke-test-http-publish-host',
```

Comment:
Why do we need a sub-project for this, what's wrong with using the existing HTTP testing sub-project?

Reply:
I wanted a cluster with two nodes each setting `publish_host`. Not sure how i can bolt that on to the the existing HTTP testing sub project as part of the `integTest` gradle configuration, happy to learn how though :smile:

Reply:
You can start a second cluster after the first cluster stops. Look at `smoke-test-client` for an example where a second cluster is started after the first cluster stops.

Reply:
I may need a little more hand holding where exactly I should add my `*IntegCluster`, will pick it up again tomorrow.

Reply:
Also, can you explain why two nodes are needed?

Reply:
I am more than happy to help, reach out tomorrow.

Reply:
I wanted two nodes to be sure no information is lost on the transport layer, since I already needed a separate cluster to boot with the `publish_host` setting this seemed like the easiest way forward. Happy to drop down to one and write an isolated unit test for binary transport. 

Will ping you today, definitely need a nudge to see where I need to be adding what 👍

Reply:
I do not think we should be testing the transport layer here, that should happen elsewhere. Let's use one node.